manifests/certonly.pp

```diff
@@ -62,9 +63,9 @@
   }
 
   if ($custom_plugin) {
-    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly "
+    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly --rsa-key-size ${key_size} "
   } else {
-    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly -a ${plugin} "
+    $command_start = "${letsencrypt_command} --text --agree-tos --non-interactive certonly -a ${plugin} --rsa-key-size ${key_size} "
```

Comment:
Nit: I'd prefer to swap `-a` and `--rsa-key-size` so the start of the line is exactly the same for both with and without a custom plugin.

Reply:
The last push passes travis ci and addresses your concern. Unless I have missed something or there is something else, please consider accepting the pull request.